Frameworks/UIKit/UITapGestureRecognizer.mm

```diff
@@ -256,12 +256,12 @@ - (void)touchesEnded:(NSSet*)touches withEvent:(UIEvent*)event {
     }
 
     id allTouches = [event allTouches];
-    unsigned count = [touches count];
 
-    if (count > _numberOfTouchesRequired || EbrGetMediaTime() - _tapTime > 0.75f) {
-        TraceVerbose(TAG, L"too many touches, numberOfTouchesRequired=%d, count=%d", _numberOfTouchesRequired, count);
+    // We may have to revisit this logic, since the UITouch tap count should drive it.
```

Comment:
> // We may have to revisit this logic, since the UITouch tap count should drive it. [](start = 3, length = 83)

Why not leverage the tap count now?